src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int randomNumber = random.nextInt(100);
+        String color = colorSupplier.getRandomColor();
+
```

Comment:
remove empty line

Reply:
Changed, but cannot find any rule for that. Seems kind of personal preferences and someone could say "Add new line before" :) Or there is some coding standard in java for this?